qutebrowser/config/parsers/keyconf.py

```diff
@@ -25,7 +25,7 @@
 
 from PyQt5.QtCore import pyqtSignal, QObject
 
-from qutebrowser.config import configdata, textwrapper
+from qutebrowser.config import configdata, textwrapper, config
```

Comment:
This causes a cyclic import because `config` imports `keyconf`, which in turn imports `config` again :wink:

You could move the `config` import inside `_validate_command`, or (slightly cleaner I think?) move the `keyconf` import inside `config` inside `_init_key_config`.